### PR TITLE
Remove links to additional elements in the PDF render

### LINKED ISSUE #162: Remove link to elements in PDF render
The link to additional elements does not resolve in the PDF render

### PR DESCRIPTION
**Pull request by palemieux**
Closes #162

### CODE REVIEW

**Review by SteveLLamb — COMMENTED** [review]
Rather than removing the link altogether (as show in the HTML as `(link)`), shouldn't the actual relative link be spelled out in PDF, such as `./elements/sample.txt` so the reader knows where to find the file in the zap package?

**Comment by SteveLLamb** [comment]
`a. Sample text element (filename: sample.txt)`

How about having the full relative path:
`a. Sample text element (filename: ./elements/sample.txt)`

And the HTML vs PDF rendering just turns on/off the `<a>` around the path itself, rather than rendering the `[link]` text?

**Comment by SteveLLamb** [comment]
> That is significantly more difficult to do since it needs to be done in CSS alone

Gotcha. Does this also do the same thing for additional elements pointing to repos or non-relative links?

**Comment by palemieux** [comment]
> > That is significantly more difficult to do since it needs to be done in CSS alone
> 
> Gotcha. Does this also do the same thing for additional elements pointing to repos or non-relative links?

Yes.

Let me try something slightly different.

File: smpte.js [hunk 1]
```diff
@@ -518,10 +518,17 @@ function insertElementsAnnex(docMetadata) {
...
 
-    e.innerText = "(link)";
+    e.innerText = "[link]";
+
+    let linkText;
+    
+    if (e.getAttribute("href").startsWith("http"))
+      linkText = `url: ${e.getAttribute("href")}`;
+    else
+      linkText = `filename: ${e.getAttribute("href").split('\\').pop().split('/').pop()}`;
```
[the remark this answers is not in the document]
Comment (palemieux) [reply smpte.js]: I do not think the relative path is useful

[the remark this answers is not in the document]
Comment (SteveLLamb) [reply smpte.js]: All good.

**Review by SteveLLamb — APPROVED** [review]

**Pull request merged**